railties/test/application/configuration_test.rb

```diff
@@ -3819,6 +3826,74 @@ class Post < ActiveRecord::Base
       assert_not_includes ActiveRecord::Base.filter_attributes, :content
     end
 
+    test "ActiveRecord::Encryption.config is ready for encrypted attributes when app is lazy loaded" do
```

Comment:
To clarify, this test is passing before the fix but is added to demonstrate both cases are handled (eager loading and lazy loading).